libevm/README.md

```diff
@@ -0,0 +1,19 @@
+# libevm
+
+libevm implements a shared library to access a standalone instance of the go-ethereum EVM and its state storage layer StateDB and underlying LevelDB.
+
+For simplicity all exported library functions take one parameter and return one value, which are all typed as C-strings and contain JSON.
+
+## Build
```

Comment:
Let's add the comment about the golang min version to be able to build

Reply:
done